.github/pull_request_template.md

```diff
@@ -7,18 +7,53 @@ v    If your PR doesn't close an issue, that's OK!  Just remove the Closes: #XXX
 
 Closes: #XXX
 
-## Description
+## What is the purpose of the change
 
-<!-- Add a description of the changes that this PR introduces and the files that
-are the most critical to review.
--->
+*Add a description of the overall background and high level changes that this PR introduces *
 
-______
+*(E.g.: This pull request improves documation of area A by adding ....*
 
-For contributor use:
 
-- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
-- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
-- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
-- [ ] Re-reviewed `Files changed` in the Github PR explorer
+## Brief change log
 
+*(for example:)*
+ 
+  - *The metadata is stored in the blob store on job creation time as a persistent artifact*
+  - *Deployments RPC transmits only the blob storage reference*
+  - *Daemons retrieve the RPC data from the blob cache*
+
+
+## PR Formatting
+
+
+- Is PR targeted against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))  (yes / no)
```

Comment:
instead of (yes/now) can we make this:
- [ ] Is PR targeted against correct branch?

So that the author can quickly toggle the desired checkboxes